sdk/python/lib/pulumi/x/automation/_cmd.py

```diff
@@ -46,7 +46,7 @@ def _run_pulumi_cmd(args: List[str],
     # All commands should be run in non-interactive mode.
     # This causes commands to fail rather than prompting for input (and thus hanging indefinitely).
     args.append("--non-interactive")
-    env = os.environ.copy().update(additional_env)
+    env = {**os.environ.copy(), **additional_env}
```

Comment:
Is the `copy()` necessary now? Wondering if it could be:

```suggestion
    env = {**os.environ, **additional_env}
```

Reply:
Good call!